argilla/docs/how_to_guides/index.md

```diff
@@ -5,7 +5,9 @@ hide: toc
 
 # How-to guides
 
-These are the how-to guides for *the Argilla SDK*. They provide step-by-step instructions for common scenarios, including detailed explanations and code samples.
+These are the how-to guides for *the Argilla SDK*. They provide step-by-step instructions for common scenarios, including detailed explanations and code samples. We have divided the guides into two categories: basic and advanced. The basic guides will help you get started with the core concepts of Argilla, while the advanced guides will help you explore more advanced features.
+
+## Basic
```

Comment:
I'm still unsure about adding two levels here, I assume this has been discussed with @burtenshaw but I thinking adding two levels adds complexity for finding key information. That said, is this PR building docs I can check?

Reply:
I was not sure either. Briefly discussed this yesterday with @burtenshaw. There is not direct level difference but only a non-clickable header as separation but feel free to take a look.

@dvsrepo There are always hidden docs being build whenever the branch contains docs/. This gets messaged like this: https://github.com/argilla-io/argilla/pull/5189#issuecomment-2217788221